vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -236,107 +228,178 @@ An `env.json` file to configure the application URLs and client credentials must
 |  ENTANDO_APPNAME | stores the Entando Application name related to the current profile |
 |  DESIGNATED_JAVA_HOME | stores the path of the Java version used internally by ent |
 
-## Diagnostic Commands
-Performing the [Automatic Install](../../docs/getting-started/README.md#automatic-install) found in the Getting Started guide installs the CLI in a Multipass VM. The following commands can be run from this VM for insight into an Entando Application.
+## Diagnostics and Debug
+
+### Diagnostic Commands
+
+Diagnostic information can be highly useful when analyzing and troubleshooting behavior. The Getting Started guide includes steps to [automatically install](../../docs/getting-started/README.md#automatic-install) the CLI in a Multipass VM. The following commands can be run from this VM for insight into an Entando Application.
+
+#### Basic Information
 
-1. Display basic information about Kubernetes and Entando resources (e.g. namespace, pods, ingresses)
+Display basic information about Kubernetes and Entando resources (e.g. namespace, pods, ingresses):
 ``` sh
 ent app-info
 ```
 
-2. Display `kubectl describe` and `kubectl logs` for each of the major Entando pods in a namespace
+#### Pod Logs
+
+Display `kubectl describe` and `kubectl logs` for each of the major Entando pods in a namespace:
 ``` sh
 ent pod-info
 ```
 
-3. List the current pods in an Entando namespace and prepare a diagnostic tar.gz 
+#### Diagnostic Files
+
+List the current pods in an Entando namespace and prepare a diagnostic tar.gz:
 ``` sh
 ent diag
 ```
-This outputs `kubectl describe` and `kubectl logs` for each of the fundamental Entando pods. It also exports custom resources, ingresses, deployments, "previous" pod logs, namespace events, etc. Diagnostic information can be highly useful when analyzing and troubleshooting behavior. The command `ent diag` outputs the directory paths of the diagnostic logs, similar to the following
+This outputs `kubectl describe` and `kubectl logs` for each of the fundamental Entando pods. It also exports custom resources, ingresses, deployments, "previous" pod logs, namespace events, etc.  
+
+The directory paths of the diagnostic logs are listed, similar to the following:
 
 ```
 > Collected diagdata available under "~/.entando/reports/entando-diagdata-2020-11-19T02:58:47+0000" for consultation
 > Collected diagdata available in archive "~/.entando/reports/entando-diagdata-2020-11-19T02:58:47+0000.tgz"
 ```
+### Troubleshooting
+The CLI is a useful debug tool to identify and correct issues with an Entando instance. 
+
+#### Error Location
+Zero in on error messages in the EntandoApp pods:
+``` bash
+ent pod grep --all "error|fail"
+```
+or
+``` bash
+ent pod grep ".*" "error|fail"
+```
+#### Extraction Error
+To resolve the error "Unable to extract the application client secret":
+
+1. Verify that the current profile namespace and application name are correct match the output of the following command
```

Comment:
and application name are correct match the output >> and application name match the output with the following command??

what is it correcting? a little unclear